shared/templates/firefox_policy-setting/template.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import print_function
+import re
+import json
+
+
+def normalize_policy_path(path_string):
+    """Generate a normalized path array from yaml.
+       Paths are space-separated key names.
+       All firefox policies are under the policies key.
+    """
+    path = []
+    if len(path_string.strip()) > 0:
+        path = re.split(" ", path_string)
+    if 'policies' not in path:
```

Comment:
Shouldn't it be `if path[0] != 'policies'`?

Reply:
I consider them equivalent and just naturally went to the "if not in X". Switching to the other form is fine.

Reply:
https://github.com/ComplianceAsCode/content/pull/8271/commits/879ea5ffb21522c9f3a1b64782d78673eda36c6e